src/main.rs

```diff
@@ -422,40 +393,85 @@ fn remote_server_options(matches: &ArgMatches) -> Result<(Option<Sources>, Optio
     Ok((sources, docker_config))
 }
 
-fn verification_options(
+fn verification_options(matches: &ArgMatches) -> Result<Option<VerificationSettings>> {
+    let verification_settings = if let Some(verification_config_path) =
+        matches.value_of("verification-config-path")
+    {
+        Some(read_verification_file(Path::new(
+            &verification_config_path,
+        ))?)
+    } else {
+        let verification_config_path = DEFAULT_ROOT.config_dir().join("verification-config.yaml");
+        if Path::exists(&verification_config_path) {
+            Some(read_verification_file(&verification_config_path)?)
+        } else {
+            build_verification_settings_from_flags(matches)?
+        }
```

Comment:
I would give higher precedence to the cli flags provided by the user. Meaning: if the user provides pub keys/annotations constraints the configuration file on the disk is not going to be read.

Reply:
True! will do.